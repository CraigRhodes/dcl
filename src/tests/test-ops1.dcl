int main()
{
  print_line(1 + 2);
  print_line(1 - 2);
  print_line(1 * 2);
  print_line(100 / 2);
  print_line(99);
  print_bool(1 == 2);
  print_bool(1 == 1);
  print_line(99);
  print_bool(1 != 2);
  print_bool(1 != 1);
  print_line(99);
  print_bool(1 < 2);
  print_bool(2 < 1);
  print_line(99);
  print_bool(1 <= 2);
  print_bool(1 <= 1);
  print_bool(2 <= 1);
  print_line(99);
  print_bool(1 > 2);
  print_bool(2 > 1);
  print_line(99);
  print_bool(1 >= 2);
  print_bool(1 >= 1);
  print_bool(2 >= 1); 
  return 0;
}
