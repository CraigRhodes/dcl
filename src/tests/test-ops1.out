3
-1
2
50
99
99
99
99
99
99
